lib/crack.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'enigma.rb'
+
+handle = File.open(ARGV[0], "r")
+
+incoming_message = handle.read.to_s.chomp
+
+handle.close
+
+enigma = Enigma.new
+translated = enigma.crack(incoming_message, ARGV[2])
```

Comment:
Might want to make this match the others in final refactoring - ARGV[2] vs. creating a new variable name